build.fsx

```diff
@@ -1,3 +1,5 @@
+System.IO.Directory.SetCurrentDirectory __SOURCE_DIRECTORY__
+
```

Comment:
ideally setting the directory should be the first thing that you do, i changed the way it's done because the old environment way doesn't work on netcore